wondrous-app/components/organization/wrapper/BoardPageHeader.tsx

```diff
@@ -207,14 +209,12 @@ function BoardPageHeader(props) {
       )}
       <ContentContainer>
         <TokenHeader>
-          <HeaderMainBlock>
-            <HeaderTopLeftContainer>
-              <HeaderTitle>{ENTITIES_DISPLAY_LABEL_MAP[entity?.toString()] || headerTitle}</HeaderTitle>
-              <PrivacyContainer>
-                <PrivacyText>{orgData?.privacyLevel !== PRIVACY_LEVEL.public ? 'Private' : 'Public'}</PrivacyText>
-              </PrivacyContainer>
-            </HeaderTopLeftContainer>
-          </HeaderMainBlock>
+          <Grid container width="fit-content" alignItems="center" gap="8px">
```

Comment:
doesnt changing this also change other pages?

Reply:
Yes, I updated it to include the other pages. I asked Ben about it. https://www.figma.com/file/UQAMcM7koodIoq2OTID7tj?node-id=23453:294651#374439477